### DISCUSSION TITLE
how to connect the same user's different devices? what are the best practices?

### DISCUSSION THREAD

**Discussion by ghost** [socketio/socket.io#4784] [Q&A]

**Comment by darrachequesne** [comment]
Hi! One common way is to use the user ID as a room, so that you can notify the user with `io.to(<user-id>).emit(...)`.

Reference: https://socket.io/docs/v4/rooms/

> I would highly appreciate if you give me sample todo app code which user's changes on todos & dark/light theme will be shared between react web app and react native mobile app via socket.io. and backend express. Thanks:)

I will get back to you.
